src/StreamDefinition.cpp

```diff
@@ -108,7 +108,7 @@ void StreamDefinition::addTrack(const uint64_t track_id,
                                 MKV_TRACK_INFO_TYPE track_type,
                                 const uint8_t* codecPrivateData,
                                 uint32_t codecPrivateDataSize) {
-    stream_info_.streamCaps.frameOrderingMode = FRAME_ORDERING_MODE_MULTI_TRACK_AV_COMPARE_PTS_ONE_MS_COMPENSATE;
+    stream_info_.streamCaps.frameOrderingMode = FRAME_ORDERING_MODE_MULTI_TRACK_AV_COMPARE_PTS_ONE_MS_COMPENSATE_EOFR;
```

Comment:
These modes are inscrutable for customers :(
Not needed for this PR, but eventually we should remove frameOrderingMode altogether as a customer controlled setting.

Reply:
Here we actually hard-code the value rather let the application supply it

Reply:
Right in this PR I'm just updating the hardcoded value but I think @sharadrj is talking about just a few lines down we do have a public API method:      

void setFrameOrderMode(FRAME_ORDER_MODE mode);

We should remove this in the future perhaps in favor of a more user understandable enum which we can document an then internally we can use that value to pick the correct frame order mode.